script/prune-dts.ts

```diff
@@ -0,0 +1,314 @@
+// https://github.com/firebase/firebase-js-sdk/blob/master/repo-scripts/prune-dts/prune-dts.ts
```

Comment:
How about talking about the content of the Redistribution section in the [apache LICENSE-2.0](https://www.apache.org/licenses/LICENSE-2.0.html) documentation?
Reading that section, it seems that license notation, correction notice, etc. are missing.
I don't know much about licensing, so I'd like to hear from you guys.

Reply:
I think it would be nice to add comments like [Firebase](https://github.com/firebase/firebase-js-sdk/blob/5ad7ff2ae955c297556223e6cb3ad9d4b897f664/packages/auth/buildtools/sauce_connect.sh#L22-L23).

/*
 * Copyright 2020 The Yorkie Authors. All rights reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 * 
 * Script copied from the Firebase JS SDK repository:
 * https://github.com/firebase/firebase-js-sdk/blob/master/repo-scripts/prune-dts/prune-dts.ts
 */